securesignals-ima-devapp/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.UID2AndroidSDK" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
Let's remove the custom theme, I don't think it's required.

Reply:
Done.

Reply:
I spoke too soon.  AppCompatActivity throws an error if there's no Theme.  I'm going to leave it in.